packages/app/src/components/challenges/Confirm.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { ReactElement } from 'react';
+import { Text, View, StyleSheet } from 'react-native';
+import { useMutation } from 'relay-hooks';
+import { Button } from '../UI';
+import {
+    CreateChallenge,
+    optimisticUpdater,
+    updater
+} from './mutations/CreateChallengeMutation';
+import {
+    CreateChallengeMutation,
+    CreateChallengeMutationResponse
+} from './mutations/__generated__/CreateChallengeMutation.graphql';
+import { FormType } from './CreateChallengeScreen';
+
+type Props = {
+    index: number;
+    setIndex: (n: number) => void;
+    form: FormType;
+    jumpTo: (s: string) => void;
+};
+
+export const Confirm = ({
+    index,
+    setIndex,
+    form,
+    me,
+    jumpTo
+}: Props): ReactElement => {
+    const [createChallenge, { loading }] = useMutation<CreateChallengeMutation>(
+        CreateChallenge
+    );
+
+    const onSubmit = () => {
+        const onError = () => {
+            console.log('onErrorCreateChallenge');
+        };
+
+        const config = {
+            variables: { input: form },
+            updater: updater(me.id),
+            optimisticUpdater: optimisticUpdater(form, me),
+            onCompleted: ({
+                CreateChallenge: { challengeEdge, error }
+            }: CreateChallengeMutationResponse) => {
+                console.log('challengeEdge: ', challengeEdge);
+            }
+        };
+        createChallenge(config);
+        setIndex(0);
+        jumpTo('live');
+    };
```

Comment:
NIT: We should also show a loader component while the user is creating the challenge. This can be pushed to a future PR.